src/sajari.js

```diff
@@ -23,13 +23,32 @@ const makeRequest = (address, body, callback) => {
     if (request.readyState !== 4) return;
 
     if (request.status === 200) {
-      callback(null, JSON.parse(request.responseText));
-    } else {
+      try {
+        callback(null, JSON.parse(request.responseText));
```

Comment:
Doesn't this try-catch also apply to errors thrown from within the callback?  If you're wanting to catch errors from JSON.parse it should probably be seperate?